<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create3dDialog</class>
 <widget class="QDialog" name="create3dDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>货物长度(dm):</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>货物高度(dm):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>栈板长度(dm):</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="goods_height_LineEdit"/>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="goods_width_LineEdit"/>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>货物宽度(dm):</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLineEdit" name="totalweightLineEdit"/>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>超托率(0~1):</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>栈板高度限制(dm):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>栈板宽度(dm):</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLineEdit" name="outRatioLineEdit"/>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>单货物重量(kg):</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="goods_length_LineEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="heightLineEdit"/>
   </item>
   <item row="17" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>总货物重量(kg):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="widthLineEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lengthLineEdit"/>
   </item>
   <item row="16" column="1">
    <widget class="QLineEdit" name="weightLineEdit"/>
   </item>
   <item row="19" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>create3dDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>create3dDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
